scripts/emulate_app.sh

```diff
@@ -177,35 +178,43 @@ if [ ! -d "./android/app" ]; then
     echo "Try './scripts/$(basename $0)' from zingo-mobile root directory." >&2
     exit 1
 fi
-cd android
 
-echo -e "\nInstalling latest build tools, platform tools, and platform..."
-sdkmanager --install 'build-tools;33.0.2' platform-tools
+cd android
+# Kill all emulators
+../scripts/kill_emulators.sh
 
-echo "Installing latest emulator..."
-sdkmanager --install emulator --channel=0
+test_report_dir="app/build/outputs/emulate_app_reports/${abi}"
+rm -rf "${test_report_dir}"
+mkdir -p "${test_report_dir}"
```

Comment:
why is this moved out of the test and lost its comment? i assume its to do with adding the emulator printouts when creating a snapshot. see below